metaflow/metaflow_config.py

```diff
@@ -199,3 +199,14 @@ def get_authenticated_boto3_client(module, params={}):
                 raise MetaflowException(repr(e))
         return boto3.session.Session(**cached_aws_sandbox_creds).client(module, **params)
     return boto3.client(module, **params)
+
+# Check if there is a an extension to Metaflow to load and override everything
+try:
+   import metaflow_custom.config.metaflow_config as extension_module
```

Comment:
Indentation seems slightly off in this section of code. Could you double check?

Reply:
Fixed. Good catch.